client/src/templates/Challenges/components/output.css

```diff
@@ -20,5 +20,5 @@ pre.output-text code {
 }
 
 .output-text:focus:not(:focus-visible) {
-  outline: none;
+  outline-color: transparent;
```

Comment:
```suggestion
```

Revert back to `outline: none`. This is for non-keyboard focus in which we don't want a focus outline.